src/views/StandaloneDirectorsFiling.vue

```diff
@@ -738,76 +750,77 @@ export default {
-              }
-            }
+        const filing = response?.data?.filing
+
+        // verify data
+        if (!filing) throw new Error('Missing filing')
+        if (!filing.header) throw new Error('Missing header')
+        if (!filing.business) throw new Error('Missing business')
+        if (filing.header.name !== FilingTypes.CHANGE_OF_DIRECTORS) throw new Error('Invalid filing type')
+        if (filing.business.identifier !== this.entityIncNo) throw new Error('Invalid business identifier')
+        if (filing.business.legalName !== this.entityName) throw new Error('Invalid business legal name')
+
+        // load Certified By (but not Date)
+        this.certifiedBy = filing.header.certifiedBy
+
+        // load Staff Payment properties
+        if (filing.header.routingSlipNumber) {
+          this.staffPaymentData = {
+            option: StaffPaymentOptions.FAS,
+            routingSlipNumber: filing.header.routingSlipNumber,
+            isPriority: filing.header.priority
+          }
+        } else if (filing.header.bcolAccountNumber) {
+          this.staffPaymentData = {
+            option: StaffPaymentOptions.BCOL,
+            bcolAccountNumber: filing.header.bcolAccountNumber,
+            datNumber: filing.header.datNumber,
+            folioNumber: filing.header.folioNumber,
+            isPriority: filing.header.priority
+          }
+        } else if (filing.header.waiveFees) {
+          this.staffPaymentData = {
+            option: StaffPaymentOptions.NO_FEE
+          }
+        } else {
+          this.staffPaymentData = {
+            option: StaffPaymentOptions.NONE
+          }
+        }
 
-            if (filing.header.effectiveDate) {
-              const effectiveDate = this.convertUTCTimeToLocalTime(filing.header.effectiveDate).slice(0, 10)
-              // restore date in COD Date component
-              this.initialCODDate = effectiveDate
-              // restore Draft Date in Directors List component
-              // FUTURE: use props instead of $refs (which cause an error in the unit tests)
-              if (this.$refs.directorsList?.setDraftDate) {
-                this.$refs.directorsList.setDraftDate(effectiveDate)
-              }
-            } else {
-              // eslint-disable-next-line no-console
-              console.log('fetchChangeOfDirectors() error = missing Effective Date')
-            }
+        if (filing.header.effectiveDate) {
+          const effectiveDate = this.convertUTCTimeToLocalTime(filing.header.effectiveDate).slice(0, 10)
+          // restore date in COD Date component
+          this.initialCODDate = effectiveDate
+          // restore Draft Date in Directors List component
+          // FUTURE: use props instead of $refs (which cause an error in the unit tests)
+          if (this.$refs.directorsList?.setDraftDate) {
+            this.$refs.directorsList.setDraftDate(effectiveDate)
+          }
+        } else if (filing.header.date) {
```

Comment:
I added this fallback logic -- if "effective date" isn't present then try to use "filing date" (which was set to current date when this draft was saved).

There's a similar sort of thing in the AR filing: if there is no effective data then fall back to AGM date.